perf/benchmark/runner/runner.py

```diff
@@ -199,7 +201,8 @@ def execute_sidecar_mode(self, sidecar_mode, load_gen_type, load_gen_cmd, sideca
                 self.mesh,
                 self.server.name,
                 labels + perf_label_suffix,
-                duration=40)
+                duration=self.duration,
```

Comment:
on L195 we `kubectl exec fortio ...` which runs synchronously for self.duration. Then here we run `perf` for self.duration. Is this happening *after* we complete the load test? Don't we need it during the load test?

Reply:
yes, it happens after we run the load test, the load test is using the duration value that user passed, which is self.duration. So that perf test should use the same thing, not 40s.

Reply:
doesn't perf need to run *while* the test is occurring though, not afterwards?

Reply:
Good catch!!! 
it actually a duplicate run, for perf test, which is running twice.

say for bothmode, we run bothmode, then we run bothmode for perf. Yes, I agree, we should combine the two. Currently, when we set --perf flag, we run twice, which is not the ideal behaviour. I will made the fix in #992 

This PR I only want to introduce the --frequency flag.